### PR TITLE
[Dashboard] Fix newIA not respecting the "DISABLED" flag for Grafana

### PR DESCRIPTION
**Pull request by alanwguo**
## Why are these changes needed?

Also adjust the color of the Grafana warning to be "info" instead of "warning"

Disabled:
Screenshot 2023-03-22 at 12 27 07 PM

Grafana not setup:
Screenshot 2023-03-22 at 3 50 36 PM

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by alanwguo** [comment]
> Shall we also simplify the instructions on the card? Something like this? image

Sounds good. I'll put this message in both the front page and the metrics page.

For the front page, it'll be this blue color. For the metrics page, I'll keep it as a "warning". The idea being the front page is simply a suggestion vs if you hit the metrics page and expect this to work, something is broken.

**Comment by scottsun94** [comment]
Sounds good!

File: dashboard/client/src/pages/metrics/Metrics.tsx [hunk 1]
```diff
@@ -299,24 +299,42 @@ export const Metrics = () => {
...
   return grafanaHost === undefined || !prometheusHealth ? (
-    <Alert className={className} severity="warning">
-      Grafana or prometheus server not detected. Please make sure both services
-      are running and refresh this page. See:{" "}
+    <Alert className={className} severity={severity}>
+      <span className={classes.heading}>
+        Set up Prometheus and Grafana for better Ray Dashboard experience
+      </span>
+      <br />
+      <br />
```
Comment (scottsun94) [inline dashboard/client/src/pages/metrics/Metrics.tsx:327]:
```suggestion
```
Let's remove one of them? the spacing looks a bit too much.

Comment (scottsun94) [reply dashboard/client/src/pages/metrics/Metrics.tsx:327]: nvm. Just tested it. Removing it gets rid of all the space. Let's keep it as is for now for simplicity.

File: dashboard/client/src/pages/layout/MainNavLayout.tsx [hunk 2]
```diff
@@ -173,13 +174,19 @@ const MainNavBar = () => {
   const classes = useMainNavBarStyles();
   const { mainNavPageHierarchy } = useContext(MainNavContext);
   const rootRouteId = mainNavPageHierarchy[0]?.id;
+  const { metricsContextLoaded, grafanaHost } = useContext(GlobalContext);
+
+  let navItems = NAV_ITEMS;
+  if (!metricsContextLoaded || grafanaHost === "DISABLED") {
```
[the remark this answers is not in the document]
Comment (alanwguo) [reply dashboard/client/src/pages/layout/MainNavLayout.tsx:180]: only the case where it's intentionally disabled.

**Review by rkooo567 — APPROVED** [review]

**Pull request merged**